ext/opentelemetry-ext-requests/tests/test_requests_integration.py

```diff
@@ -229,3 +230,10 @@ def test_custom_tracer_provider(self):
         span = span_list[0]
 
         self.assertIs(span.resource, resource)
+
+    def test_if_headers_equals_none(self):
+        RequestsInstrumentor().uninstrument()
+        RequestsInstrumentor().instrument(tracer_provider=TracerProvider())
```

Comment:
what purpose does clearing the instrumentor serve? by default the instrumentor is turned on, so I imagine the error would surface regardless.

Reply:
agreed with you, removed this

Reply:
after removing this the pypy tests failed, i repeated logic of previous tests, i think that this used as preparing test environment somehow

Reply:
deleted this redudant instrumentation